yellowbrick/text/tsne.py

```diff
@@ -335,7 +345,7 @@ def draw(self, points, target=None, **kwargs):
         for label, points in series.items():
             self.ax.scatter(
                 points['x'], points['y'], c=colors[label],
-                alpha=0.7, label=label
+                alpha=self.alpha, label=label
             )
 
     def finalize(self, **kwargs):
```

Comment:
If we're allowing the opacity to change, do we also want to use `manual_legend` here?

Reply:
Another great point! I have approached this by changing `color_values` (which is created in `draw` using the `resolve_colors` method) into `self.color_values_` so that it can be used by `manual_legend` in `finalize`.